frontend/public/components/dashboard/inventory-card/inventory-item.tsx

```diff
@@ -110,6 +121,7 @@ export const ResourceInventoryItem: React.FC<ResourceInventoryItemProps> = React
             count={groups[key].count}
             statusIDs={groups[key].statusIDs}
             filterType={groups[key].filterType}
+            showLink={showLink}
```

Comment:
if you dont want to show link, then you should use `Status` component instead of `StatusLink`

Reply:
Well in that case I think I should just make use of that component than making changes to StatusLink. Sorry for the trouble.